app/scripts/modules/kubernetes/src/rawResource/component/group/RawResource.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { UISref, UISrefActive } from '@uirouter/react';
+import { CloudProviderLogo } from '@spinnaker/core';
+import './RawResource.less';
+import { RawResourceUtils } from '../RawResourceUtils';
+
+interface IRawResourceProps {
+  resource: IApiKubernetesResource;
+}
+
+interface IRawResourceState {}
+
+export class RawResource extends React.Component<IRawResourceProps, IRawResourceState> {
+  constructor(props: IRawResourceProps) {
+    super(props);
+  }
+
+  public render() {
+    const key =
+      this.props.resource.account +
+      '-' +
+      this.props.resource.namespace +
+      '-' +
+      this.props.resource.kind +
+      '-' +
+      this.props.resource.displayName;
+    const params = {
+      account: this.props.resource.account,
+      name: this.props.resource.name,
+      region: this.props.resource.region,
+    };
```

Comment:
A little syntax sugar, in case you didn't know it:

```suggestion
    const { account, name, region, namespace, kind, displayName } = this.props.resource;
    const key = `${account}-${namespace}-${kind}-${displayName};
    const params = { account, name, region };
```